kamon-statsd/src/main/resources/reference.conf

```diff
@@ -5,18 +5,10 @@
 kamon {
   statsd {
 
-    # Hostname and port in which your StatsD is running. Remember that StatsD packets are sent using UDP and
-    # setting unreachable hosts and/or not open ports wont be warned by the Kamon, your data wont go anywhere.
-    hostname = "127.0.0.1"
-    port = 8125
```

Comment:
I see what you mean, but custom senders will be a very special situation. Out of the box we should support UDP/TCP and Batched/Unbatched senders to a single StatsD instance and for most users that will be enough. So, to keep it simple for users I would still prefer to use the old `kamon.statsd.[hostname|port]` keys that will be read by the senders provided out-of-the-box, and then custom senders (like yours) have the choice of using that or reading that info from somewhere else, as you already do.

@dpsoft what are your thoughts on this too?